### DISCUSSION TITLE
`export function load()` → `export default function`?

### DISCUSSION THREAD

**Discussion by ehzro3** [sveltejs/kit#6016] [General]
Not a big difference but I wonder if it could be more ergonomic to lose a the function name. Because `load` in both `+page|layout.js` and `+page|layout.server.js`, have taken more representative position. And I think no one will ever need to use `load` variable to call it?

```diff
- export function load() {
+ export default function () {
  // ...
}
```

What do you think?

**Comment by vekunz** [comment]
This would not work properly with TypeScript, I think:

```
export const load: Load = () => {
  // ...
};
```
